layouts/New/layouts.module.css

```diff
@@ -0,0 +1,62 @@
+.baseLayout {
+  @apply grid
+    h-screen
+    w-screen
+    grid-cols-[1fr]
+    grid-rows-[auto_1fr_auto];
+}
+
+.articleLayout {
+  @apply grid
+    grid-rows-[1fr]
+    overflow-hidden
+    grid-areas-[sidebar_main_metabar,sidebar_footer_metabar]
+    sm:grid-cols-[auto_1fr_auto]
+    xl:grid-cols-[320px_1fr_320px]
+    xs:block
+    xs:overflow-auto;
+
+  > *:nth-child(1) {
```

Comment:
I'm not a fan of this in this specific scenario. Because the article layout is generic, and simply creates 3 different containers.

left-side main content right-sie (through a grid system); So these should work regardless of who imports the article layout. Adding class names requires downstream to import the styles of the article layout, which is not what we want.

Reply:
It looks a bit dangerous to me because the order of elements is a very relative concept and easily can ruin the layout of the page. But I got your point, maybe we will consider testing it in the future 👀